app/assets/stylesheets/devise.scss

```diff
@@ -1,10 +1,10 @@
-.sign-in-modal {
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  width: 90%;
-  max-width: 500px;
+.center-content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 100vw;
+  height: 100vh;
```

Comment:
The `registrations#edit` page still doesn't work.  Make this `min-height` to fix it.

Reply:
I can make that change no problem, I don't know what page doesn't work. I'll push a commit to solve it nonetheless.